jekyll/_cci2/language-php.md

```diff
@@ -56,7 +56,7 @@ version: 2
 jobs: # a collection of steps
   build: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
-      - image: circleci/php:7.1-node-browsers # ...with this image as the primary container; this is where all `steps` will run
+      - image: cimg/php:8.0.14-node # ...with this image as the primary container; this is where all `steps` will run
```

Comment:
This line below is no longer valid in next-gen images:

```
- run: sudo docker-php-ext-install zip
```

1. The zip extension is already installed. If we just wanted to show an example of how to install something else, we would use `pecl`.

Reply:
To clarify, do you mean `sudo docker-php-ext-install pecl` ?